apps/dashboard/app/views/dashboard/_xdmod_widget_jobs.html.erb

```diff
@@ -1,92 +1,73 @@
 <div id="jobsPanelDiv"></div>
 
-
 <script id="jobs-template" type="text/x-handlebars-template">
-<div class="panel panel-default"><div class="panel-body">
-  <h4>{{title}}</h4>
-
-  <div class="table-responsive visible-xs-block">
-    <table class="table table-striped  table-condensed">
-      {{#each results}}
-      <tr>
-        <td>
-          <dl>
-            <dt>User</dt>
-            <dd>{{name}}</dd>
-            <dt>Job ID</dt>
-            <dd>{{local_job_id}}</dd>
-            <dt>Cluster/Resource</dt>
-            <dd>{{resource}}</dd>
-            <dt>Start</dt>
-            <dd>{{start_time}}</dd>
-            <dt>Stop</dt>
-            <dd>{{end_time}}</dd>
-            <dt>Walltime Used</dt>
-            <dd>{{walltime}}</dd>
-            <dt>CPU</dt>
-            <dd>{{cpu}}</dd>
-          </dl>
-        </td>
-      </tr>
-      {{else}}
-      <tr><td>No data available.</td></tr>
-      {{/each}}
-    </table>
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <a href="{{xdmod_url}}" class="pull-right">XDMoD <span class="fa fa-external-link-square-alt"></span></a>
+    <h3 class="panel-title">{{title}}</h3>
   </div>
 
-  <table class="table table-striped table-condensed hidden-xs">
-    <thead>
-      <tr>
-        <th>User</th>
-        <th>Job ID</th>
-        <th>Cluster/Resource</th>
-        <th>Start</th>
-        <th>End</th>
-        <th>Walltime Used</th>
-        <th>CPU</th>
-      </tr>
-    </thead>
-    <tbody>
-      {{#each results}}
-      <tr>
-        <td>{{name}}</td>
-        <td><a target="_blank" href="{{job_url}}">{{local_job_id}}&nbsp;<i style="display: inline" class="fa fa-external-link-square-alt"></i></a></td>
-        <td>{{resource}}</td>
-        <td>{{start_time}}</td>
-        <td>{{end_time}}</td>
-        <td>{{walltime}}</td>
-        <td>{{cpu_user}}</td>
-      </tr>
-      {{else}}
-      <tr><td colspan="7">No data available.</td></tr>
-      {{/each}}
-    </tbody>
-  </table>
-
   {{#if error}}
-    <div class="alert alert-danger">{{error}}</div>
-  {{/if}}
-
+    <div class="panel-body">
+      <div class="alert alert-danger">{{error}} Please ensure you are <a href="{{xdmod_url}}">logged into XDMoD first</a>, and then try again.</div>
+    </div>
```

Comment:
We can push this into another ticket, but we should probably extract the message out of the response from XDMOD and show it here. Currently I can't see the this graph, the error message from XDMOD is straight forward enough: `Request for unknown Realm: supremm` 

An authentication error message was `An error was encountered while attempting to process the requested authorization procedure.` so maybe we keep our message and check for the 401 status header?

Either way, it's just something to think about for later.